apps/src/code-studio/components/libraries/LibraryManagerDialog.jsx

```diff
@@ -92,6 +97,13 @@ export default class LibraryManagerDialog extends React.Component {
     );
   };
 
+  addLibrary = channelId => {
+    let libraryClient = new LibraryClientApi(channelId);
+    libraryClient.getLatestVersionId(versionId =>
```

Comment:
nit: consider renaming `getLatestVersionId` to something that more clearly implies asynchronicity. Maybe fetch?